sdk/eventhub/event-hubs/src/eventData.ts

```diff
@@ -111,67 +111,67 @@ export interface EventDataInternal {
   retrievalTime?: Date;
 }
 
-  /**
```

Comment:
This looks much messier because eslint/prettier corrected the extra white-space, but all I did was remove my changes.